### PR TITLE
#85: Update Expenses

### PR DESCRIPTION
**Pull request by ShashankKasera**

### CODE REVIEW

**Comment by ShashankKasera** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Pull request contains heavy changes. Please ensure that your PR is small for faster review.</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>Having a proper description on PR gives more clarity to the reviewer. Please ensure that the PR has a neatly written description explaining the purpose of this change request.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Let's also assign this pull request to someone for review.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems">Danger Kotlin</a> against 00a5eacf2cd8aa6caf73ab1ec1b58f6c7e5324c0
</p>

**Comment by mayankkasera** [comment]
Change the base branch

File: app/src/main/java/com/shashank/splitterexpensemanager/feature/addexpense/AddExpensesViewModel.kt [hunk 1]
```diff
@@ -95,6 +116,64 @@ class AddExpensesViewModel @Inject constructor(
...
+                        categoryId,
+                        amount,
+                        splitAmount,
+                        name,
+                        date,
+                        time,
+                        description
+                    )
+                )
+                Log.i("grkebgh", "updateExpenses: 1 $allOweOrOwedList $splitAmount")
```
Comment (mayankkasera) [inline app/src/main/java/com/shashank/splitterexpensemanager/feature/addexpense/AddExpensesViewModel.kt:151]: Remove unused logs

File: app/src/main/java/com/shashank/splitterexpensemanager/feature/addexpense/AddExpensesViewModel.kt [hunk 2]
```diff
@@ -95,6 +116,64 @@ class AddExpensesViewModel @Inject constructor(
...
+
+                allOweOrOwedList.forEach {
+                    Log.i("grkebgh", "updateExpenses: $allOweOrOwedList $splitAmount")
```
Comment (mayankkasera) [inline app/src/main/java/com/shashank/splitterexpensemanager/feature/addexpense/AddExpensesViewModel.kt:154]: This as well

File: app/src/main/java/com/shashank/splitterexpensemanager/feature/addexpense/AddExpensesViewModel.kt (hunk 1 above)
Comment (ShashankKasera) [reply app/src/main/java/com/shashank/splitterexpensemanager/feature/addexpense/AddExpensesViewModel.kt:151]: done

File: app/src/main/java/com/shashank/splitterexpensemanager/feature/addexpense/AddExpensesViewModel.kt (hunk 2 above)
Comment (ShashankKasera) [reply app/src/main/java/com/shashank/splitterexpensemanager/feature/addexpense/AddExpensesViewModel.kt:154]: done

**Review by mayankkasera — APPROVED** [review]

**Pull request merged**